hrjobcontract/js/src/job-contract/controllers/form/form-general.js

```diff
@@ -89,7 +89,7 @@ define([
         months = m.diff(dateStart, 'months');
 
         m.add(-months, 'months');
-        days = m.diff(dateStart, 'days');
+        days = m.diff(dateStart, 'days') + 1;
```

Comment:
Do you think you can add a test for this, to make sure we won't get a regression?